.github/workflows/linting.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Install flake8
         run: python -m pip install -r ci/linting_requirements.txt
 
+      - name: Install MetPy Flake8 Plugin
+        run: python -m pip install tools/flake8-metpy
```

Comment:
by the way, you might be interested in [flake8 local plugins](https://flake8.pycqa.org/en/latest/user/configuration.html#using-local-plugins) (then you don't need a special installation)

Reply:
That's really helpful, it definitely removed some boilerplate. Thanks!